### DISCUSSION TITLE
v0.30.1

### DISCUSSION THREAD

**Discussion by marctc** [grafana-cold-storage/agent#2650] [Announcements]
This is release `v0.30.1` of the Grafana Agent.

### Upgrading
Read the [upgrade guide](https://grafana.com/docs/agent/v0.30/upgrade-guide) for specific instructions on upgrading from older versions.

#### Bugfixes

- Fix issue where journald support was accidentally removed. (@tpaschalis)

- Fix issue where some traces' metrics where not collected. (@marctc)

### Installation:
Grafana Agent is currently distributed in plain binary form, Docker container images, a Windows installer, and a Kubernetes install script. Choose whichever fits your use-case best.

#### Kubernetes

Install directions [here.](https://grafana.com/docs/grafana-cloud/quickstart/agent-k8s/)

#### Docker container:

* https://hub.docker.com/r/grafana/agent

```bash
docker pull "grafana/agent:v0.30.1"
```

#### Windows installer

The Windows installer is provided as a [release asset](https://github.com/grafana/agent/releases/download/v0.30.1/grafana-agent-installer.exe) for x64 machines.

#### Binary

We provide precompiled binary executables for the most common operating systems. Choose from the assets below for your matching operating system.

Note: ppc64le builds are currently considered secondary release targets and do not have the same level of support and testing as other platforms.

Example for the `linux` operating system on `amd64`:

```bash
# download the binary
curl -O -L "https://github.com/grafana/agent/releases/download/v0.30.1/agent-linux-amd64.zip"

# extract the binary
unzip "agent-linux-amd64.zip"

# make sure it is executable
chmod a+x "agent-linux-amd64"
```

#### `agentctl`

`agentctl`, a tool for helping you interact with the Agent, is available as a Docker image:

```bash
docker pull "grafana/agentctl:v0.30.1"
```

Or as a binary. Like before, choose the assets below that matches your operating system. For example, with `linux` on `amd64`:

```bash
# download the binary
curl -O -L "https://github.com/grafana/agent/releases/download/v0.30.1/agentctl-linux-amd64.zip"

# extract the binary
unzip "agentctl-linux-amd64.zip"

# make sure it is executable
chmod a+x "agentctl-linux-amd64"
```

#### `agent-operator`

`agent-operator`, a Kubernetes Operator for the Grafana Agent, is available only as a Docker image:

```bash
docker pull "grafana/agent-operator:v0.30.1"
```

<hr /><em>This discussion was created from the release <a href='https://github.com/grafana/agent/releases/tag/v0.30.1'>v0.30.1</a>.</em>

**Comment by youngpabl0** [comment]
``docker pull "grafana/agent:v0.30.1"``
hello guyz, there are no docker image for agent 0.30.1 :)

**Reply by marctc** [reply]
Should work right now. Thanks for reporting!

**Comment by captncraig** [comment]
Yes, looks like a build problem with build tags. Working on it.
